src/components/ui/PriceDisplay.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class PriceDisplay extends React.Component {
+    static propTypes = {
+        fish: PropTypes.object.isRequired,
+        professionMultiplier: PropTypes.number.isRequired,
+    };
+
+    render() {
+        const { professionMultiplier, fish: {price} } = this.props;
+        const { base, silver, gold } = price;
+
+        return (
+            <React.Fragment>
+                <div>{Math.floor(base * professionMultiplier)}</div>
```

Comment:
You're doing `Math.floor(someNumber * professionMultiplier)` and awful lot - could you make this a function to cut down on repetition?

`profMultiply(base);`
`profMultiply(silver);`
etc